cypress/e2e/firstTests.cy.js

```diff
@@ -0,0 +1,76 @@
+/// <reference types="cypress" />
+
+describe('My first test', () => {
+  it('should open Sign in Page', () => {
+    cy.visit('https://react-redux.realworld.io/')
+    .get('a')
+    .should('contain.text', 'conduit');
+
+    cy.contains('Sign in')
+    .should('exist')
+    .click();
+
+    cy.url('include', '/login');
+
+    cy.get('h1')
+    .should('contain.text', 'Sign In');
+  });
+
+  it('should fill fields', () => {
+    cy.get(':nth-child(1) > .form-control')
```

Comment:
```suggestion
    cy.get(':nth-child(1) > .form-control')
```
Try to find better selectors than these :)